akka-docs/src/main/paradox/stream/operators/Source-or-Flow/conflate.md

```diff
@@ -18,6 +18,15 @@ Allow for a slower downstream by passing incoming elements and a summary into an
 there is backpressure. The summary value must be of the same type as the incoming elements, for example the sum or
 average of incoming numbers, if aggregation should lead to a different type `conflateWithSeed` can be used:
 
+## Example
+
+Scala
+:   @@snip [IntegrationDocSpec.scala]($code$/scala/docs/stream/IntegrationDocSpec.scala) { #conflate }
```

Comment:
should we standardize on where to put those snippets for the operators docs, such as in existing https://github.com/akka/akka/tree/master/akka-docs/src/test/scala/docs/stream/operators

To make it easy for contributors of those examples (because the core team will not have time to write them all) we should keep the ambition rather low. They don't have to be full tests. Functionality should be tested elsewhere.

Also, IntegrationDocSpec is about integration with actors (or outside world) so I don't think that is a good place for this one

Reply:
I see; the location for the example was based on a different operator docs; that one was not following this style as well then. I'll align them all together in this PR